pystac/item.py

```diff
@@ -28,6 +27,9 @@
 
 T = TypeVar("T", bound="Item")
 
+if TYPE_CHECKING:
+    from datetime import datetime as Datetime
```

Comment:
Any reason not to keep the lowercase version here, as done elsewhere?

```suggestion
    from datetime import datetime
```

Reply:
Oh yeah I should have mentioned this. I _did_ try changing this to `from datetime import datetime` but I ran into conflicts since there are also kwargs and attrs called `datetime` and in the file.

Reply:
Ok, can you then throw a comment on that import w/ that explanation? If we're documenting our decisions w/ a style guide we should call out deviations from that guide.